toar/toar_planetscope.ipynb

```diff
@@ -0,0 +1,474 @@
+    "\n",
+    "Top of Atmosphere Reflectance is extremely useful because it is an apples-to-apples comparable number from any satellite over any location that does not change with time of day or time of year **unless the content of the scene changes.** It is very commonly used in GIS applications which compute spectral indices such as NDVI or NDWI. \n",
+    "\n",
+    "It is so broadly useful that Planet does this math for you and provides all the coefficients necessary to convert a radiance image into a reflectance image!"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "$$\n",
+    "\\begin{equation*}\n",
+    "\\mathbf{img}_{reflectance} =  \\begin{vmatrix}\n",
+    "a \\\\\n",
+    "b \\\\\n",
+    "c \\\\\n",
+    "d \n",
+    "\\end{vmatrix}\n",
+    "\\times \\mathbf{img}_{radiance}\n",
+    "\\end{equation*}\n",
+    "$$\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The four coefficients $a, b, c, d$ are calculated and provided with every analytic image that Planet provides and can be used as simple scaling factors to convert from radiance to reflectance. Their values change with the image's local \n",
+    "time of day and time of year, and do so uniquely per satellite."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**In this guide, you'll perform a basic Radiance to Reflectance calculation on PlanetScope imagery using just a few lines of Python. Here are the steps:**\n",
+    "\n",
+    "1. Download a PlanetScope image\n",
+    "2. Extract data from each color band\n",
+    "3. Extract the coefficients\n",
+    "4. Convert Radiance to Reflectance\n",
+    "5. Save the Reflectance\n",
+    "6. Apply a color scheme to visualize reflectance on a map"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Requirements\n",
+    "- Python 2.7 or 3+\n",
+    "- [Planet's Python Client](https://www.planet.com/docs/api-quickstart-examples/cli/)\n",
+    "- [rasterio](https://github.com/mapbox/rasterio)\n",
+    "- [numpy](http://www.numpy.org/)\n",
+    "- [matplotlib](https://matplotlib.org/)\n",
+    "- [Planet API Key](https://www.planet.com/account/#/), stored as environment variable `$PL_API_KEY`.\n",
+    "- [Planet 4-Band Imagery](https://www.planet.com/docs/imagery-quickstart/) with the following specifications: `item-type`: `PSOrthoTile`, `REOrthoTile`, or `PSScene4Band`; `asset-type`: `analytic`, or `basic_analytic`"
+   ]
+  },
```

Comment:
You could skip listing the steps and requirements and tell users to follow the NDVI guide through the few steps. This guide should focus on the conversion from radiance to reflectance.